packages/Algernon.package/ALGExpressionType.class/instance/getCommand.st

```diff
@@ -1,6 +1,9 @@
 actions
 getCommand
-	| textFieldContents |
+	| textFieldContents startIndex |
 	textFieldContents := self typeObject topmostQuery.
-
-	^ textFieldContents copyFrom: 3 to: textFieldContents size"first 2 characters are do or '= '"
\ No newline at end of file
+	(textFieldContents at: 1) == $=
```

Comment:
Just some nitpicking:
`textFieldContents first = $=`
 would be a bit more squeak-like

Reply:
did that :+1:

Reply:
Thank you 👍